services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationServiceITest.kt

```diff
@@ -923,7 +924,9 @@ class ApplicationServiceITest : DatabaseTest() {
                 }
         val applicationData = application.applicationData as CableReportApplicationData
         val pendingApplicationData = applicationData.copy(pendingOnClient = false)
-        every { cableReportServiceAllu.create(pendingApplicationData.toAlluData()) } returns 26
+        val expectedAlluRequest = pendingApplicationData.toAlluData(HANKE_TUNNUS)
+        assertEquals(application.hanke.hankeTunnus, expectedAlluRequest.identificationNumber)
```

Comment:
Please test this separately, not inside the arrange phase of some other test.

Reply:
toAlluData is now tested in ApplicationDataMapperTest.